plugins/pixart-rf/fu-pxi-firmware.c

```diff
@@ -10,12 +10,15 @@
 
 #define PIXART_RF_FW_HEADER_SIZE       32 /* bytes */
 #define PIXART_RF_FW_HEADER_TAG_OFFSET 24
+/* The hpac header is start from 821st byte from the end */
+#define PIXART_RF_FW_HEADER_HPAC_POS_FROM_END 821
 
 #define PIXART_RF_FW_HEADER_MAGIC 0x55AA55AA55AA55AA
 
 struct _FuPxiFirmware {
 	FuFirmware parent_instance;
 	gchar *model_name;
+	gboolean is_hpac_flag;
```

Comment:
suggest renaming to `is_hpac`

Reply:
solved in 269968b90c62b0300ff7b0ef900798a43124ff41